### ISSUE TITLE
input[type=file] support

### ISSUE THREAD

**Issue by scriptPilot** [nolimits4web/Framework7-Vue#84]
Using file inputs results in error:

> DOMException: Failed to set the 'value' property on 'HTMLInputElement': This input element accepts a filename, which may only be programmatically set to the empty string.

I suggested the following fix:

```
if (self.type === 'select' || self.type === 'textarea') {
        delete attrs.value;
        if (self.type === 'select') {
          if (self.hasSelectModel) {
            inputEl = c('select', {attrs: attrs, on: on}, self.$slots.default);
          }
          else {
            inputEl = c('select', {attrs: attrs, on: on, domProps: {value: self.valueComputed}}, self.$slots.default);
          }
        }
        else {
          inputEl = c('textarea', {attrs: attrs, on: on, domProps: {value: self.valueComputed}}, self.$slots.default);
        }
      }

     // FIX
     else if (self.type === 'file') {      
        delete attrs.value;
        inputEl = c('input', {attrs: attrs}, self.$slots.default);
      }

      else {
        if(self.$slots.default && self.$slots.default.length > 0 || !self.type) {
          inputEl = self.$slots.default;
        }
        else {
          if (self.type === 'switch') {
            inputEl = c('f7-switch', {props: attrs, on: on});
          }
          else if (self.type === 'range') {
            inputEl = c('f7-range', {props: attrs, on: on});
          }
          else inputEl = c('input', {attrs: attrs, on: on, domProps: {value: self.valueComputed, checked: self.checkedComputed}});
        }
      }
```

But ... there is no @change event emitted anymore ... any better idea?

**Comment by macdonst** [comment]
@scriptPilot why don't you send a pull request. That way it is easier to see the changes, comment and add commits.

**Comment by scriptPilot** [comment]
Pull request sent: https://github.com/nolimits4web/Framework7-Vue/pull/86/files

**Comment by nolimits4web** [comment]
Merged #86
